engine/src/main/java/org/hibernate/search/engine/backend/metamodel/IndexDescriptor.java

```diff
@@ -52,4 +54,19 @@ public interface IndexDescriptor {
 	 */
 	Collection<IndexFieldDescriptor> staticFields();
 
+	/**
+	 * Looks up the built-in and configured analyzers available to the index represented by this descriptor.
+	 *
+	 * @param name The name of the analyzer.
+	 * @return An {@link Optional#empty() empty optional} if there is no analyzer configured with the given name.
+	 */
+	@Incubating
+	Optional<? extends AnalyzerDescriptor> analyzerDescriptor(String name);
```

Comment:
Why not just `analyzer(...)`? We don't use the `Decriptor` suffix anywhere else in the metamodel methods.

Reply:
yeah, that's because of this one

https://github.com/hibernate/hibernate-search/blob/4a8b3af583676d3f71f4475d0a8fa66b87079124/backend/lucene/src/main/java/org/hibernate/search/backend/lucene/LuceneBackend.java#L23

and since I wanted to keep the same names for backend/index descriptor ... 🙈 but with backend stuff gone it should be ok 😃